src/components/pages/Login.js

```diff
@@ -1,11 +1,75 @@
-import React from 'react';
+import React from "react";
+import { useState } from "react";
+import { Redirect } from "react-router-dom";
+import axios from "axios";
+
+function Login(props) {
+  const [credentials, changeCredentials] = useState({
+    email: "",
+    password: ""
+  });
+
+  const [status, changeStatus] = useState({
+    loading: false,
```

Comment:
It's awesome that you thought about loading indicator and error display, well done 🔥